microprofile/microprofile-rest-client-1.3/src/main/java/org/wildfly/swarm/ts/microprofile/rest/client/v13/SimpleResource.java

```diff
@@ -12,4 +12,16 @@
     public Response simpleOperation() {
         return Response.ok().entity("Hello from endpoint").build();
     }
+
+    @GET
+    @Path("/ssl-client")
+    public Response simple() {
+        return Response.ok().entity("Hello from SSL client.").build();
+    }
+
+    @GET
+    @Path("/cdi-ssl-client")
+    public Response simpleCDI() {
```

Comment:
I'd remove these 2 methods (`simple` and `simpleCDI`) and let the SSL client call `/rest/simple`. Then, you would prepend something like `"SSL client got: "` to the response the SSL client received, and send that back as your response.